src/dmglib.py

```diff
@@ -213,6 +248,59 @@ def dmg_already_attached(path: str) -> bool:
     return os.path.realpath(path) in attached_images()
 
 
+def get_dmg_mountpoints(path: str) -> dict:
```

Comment:
Currently, method names start with `dmg_` mostly. Would renaming this method to `dmg_get_mountpoints` make sense to you, to stay consistent?